projects/packages/jetpack-mu-wpcom/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
```

Comment:
In other places in the monorepo, we rely on the `@automattic/jetpack-webpack-config` package we created to properly handle things like i18n:
https://github.com/Automattic/jetpack/blob/0f7db0773644742adec8e14414338d0b19fd26de/projects/js-packages/webpack-config/README.md#L1

It may be useful here as well.